### PR TITLE
Add get_active_threads

### PR DESCRIPTION
**Pull request by jrobinsonAG**
Add get_active_threads function to filter TM1 threads which are non-idle directly via REST end-point.

### CODE REVIEW

**Comment by jrobinsonAG** [comment]
@MariusWirtz is this in line with what you were looking for in relation to #564 ?

File: TM1py/Services/MonitoringService.py [hunk 1]
```diff
@@ -27,6 +27,16 @@ def get_threads(self, **kwargs) -> List:
         response = self._rest.GET(url, **kwargs)
         return response.json()['value']
 
+    def get_active_threads(self, **kwargs):
+        '''Return a list of non-idle threads from the TM1 Server
```
Comment (MariusWirtz) [inline TM1py/Services/MonitoringService.py:31]: Triple double-quotes should be used for docstrings
https://www.python.org/dev/peps/pep-0257/

**Comment by MariusWirtz** [comment]
Thanks @jrobinsonAG 

Code looks good! I could only find a small violation on double quotes and single quotes conventions :)

**Comment by jrobinsonAG** [comment]
@MariusWirtz sorry was obviously not paying attention to my finger, pre-coffee :)

**Comment by MariusWirtz** [comment]
Thanks for the contribution! Very much appreciated

**Pull request merged**